app/models/meeting_date.rb

```diff
@@ -3,7 +3,7 @@ class MeetingDate  < ActiveRecord::Base
   has_many :courses, through: :course_meetings
 
   def month_weekday
-    first.strftime('%A %B')
+    third.strftime('%A %B')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.